### ISSUE TITLE
Enabling Spy "Your Eternal Reward" and "Wanga Prick"

### ISSUE THREAD

**Issue by unusual917** [Scags/TF2-Jailbreak-Redux#27]
How would I go about enabling "Your Eternal Reward" and "Wanga Prick" for spy? Item index 225 and 574.
I looked at the weaponblocker plugin and it seems like by default it forces stock knife right? Not that I know what I'm doing when it comes to sourcepawn lol.

**Comment by Scags** [comment]
I don’t think it removes those knives by default; pretty sure I was using the wanga prick when I was testing the other day. You may have a case of conflicting plugins, but I’ll double check. 
The only weapons that are removed by default are a few on warday. Eureka Effect and the Cloak and Dagger.

**Comment by unusual917** [comment]
ok will double check and see if anything else is causing it

**Comment by unusual917** [comment]
I had the cvar that disabled it and forgot it was there xD Thanks for the input!
